streaming/index.js

```diff
@@ -1177,7 +1177,7 @@ const startServer = async () => {
     const location = url.parse(req.url, true);
 
     req.requestId = uuid.v4();
-    req.remoteAddress = ws._socket.remoteAddress;
+    req.remoteAddress = req.headers['x-forwarded-for']?.split(',')[0].trim() || req.connection.remoteAddress;
```

Comment:
```suggestion
    req.remoteAddress = req.headers['x-forwarded-for']?.split(',')[0].trim() ?? req.socket.remoteAddress;
```

I think that'll do what you want it to, but I haven't tested it, but yeah `req` here is a `http.IncomingMessage`